content/blog/2017/04/2017-04-27-columbia.adoc

```diff
@@ -0,0 +1,21 @@
+---
+:layout: post
+:title: Calling for Columbian Jenkins users!
+:tags:
+- feedback
+- general
+:author: kohsuke
+---
+
+The Jenkins project has learned that a company is trying to register "Jenkins" as a trademark in Columbia. This is alarming for us, and we are trying to oppose it. In order to do this effectively, link:https://goo.gl/forms/Ekt6YhaaAwhrS9GT2[we need to hear from Colombian users of Jenkins].
```

Comment:
This Google form was presumably created within the CloudBees organisation — nobody else has permission to access it.